docs/build/guides/basics/automate-reset-data.mdx

```diff
@@ -0,0 +1,460 @@
+    );
+    const contractBuffer = returnContractResponse.returnValue._value._value;
+    const contractId = StellarSdk.StrKey.encodeContract(contractBuffer);
+    const contract = new StellarSdk.Contract(contractId);
+
+    // Invoke Contract
+    const invokeContractTransaction = new StellarSdk.TransactionBuilder(
+      account,
+      {
+        fee: StellarSdk.BASE_FEE,
+        networkPassphrase: networkPassphrase,
+      },
+    )
+      .addOperation(
+        contract.call("hello", StellarSdk.xdr.ScVal.scvSymbol("World")),
+      )
+      .setTimeout(30)
+      .build();
+
+    const invokeContractTx = await server.prepareTransaction(
+      invokeContractTransaction,
+    );
+    invokeContractTx.sign(deployer);
+    let invokeContractResponse = await server.sendTransaction(invokeContractTx);
+    const returnInvokeContractResponse = await getTxsReports(
+      invokeContractResponse,
+      `Invoke Contract.`,
+      server,
+    );
+
+    const functionValues = returnInvokeContractResponse.returnValue._value
+      .map((item) => item._value)
+      .filter(Boolean);
+
+    return {
+      contractId: contractId,
+      returnValues: functionValues,
+    };
+  } catch (error) {
+    console.error("Error in contract deployment and invocation:", error);
+    throw error;
+  }
+}
+
+async function automateSetup() {
+  try {
+    // const networkRPC = "USE EITHER FUTERNET OR TESTNET RPC"
+    const networkRPC = "https://soroban-testnet.stellar.org";
+    // Example
+    // FOR FUTERENET - https://rpc-futurenet.stellar.org
+    // FOR TESTNET - https://soroban-testnet.stellar.org
+
+    //Initiate Server
+
+    const server = new StellarSdk.SorobanRpc.Server(networkRPC);
+
+    //Check Network Status
+    console.log("Checking network health...");
+    const health = await server.getHealth();
+    console.log("Network health:", health);
+
+    // const networkURL = "USE EITHER FUTERNET OR TESTNET URL"
+    const networkURL = "https://friendbot.stellar.org";
+    // Example
+    // FOR FUTERENET - https://friendbot-futurenet.stellar.org
+    // FOR TESTNET - https://friendbot.stellar.org
+
+    const accountOne = await createAccount(networkURL);
+    const accountTwo = await createAccount(networkURL);
+
+    // const networkParaphrase = "USE EITHER FUTERNET OR TESTNET Paraphrase"
+    const networkPassPhrase = StellarSdk.Networks.TESTNET;
+    // Example
+    // FOR FUTERENET - StellarSdk.Networks.FUTURENET
+    // FOR TESTNET -  StellarSdk.Networks.TESTNET
+
+    console.log("Issuing an Asset...");
+    // Issue assets to these accounts
+    const customAsset = new StellarSdk.Asset("Boya", accountOne.publicKey());
+    await issueAsset(
+      server,
+      networkPassPhrase,
+      accountOne,
+      accountTwo,
+      customAsset,
+    );
+
+    // Create liquidity pool
+    console.log("Creating liquidity pool...");
+    const nativeAsset = StellarSdk.Asset.native();
+    await createLiquidityPool(
+      server,
+      accountOne,
```

Comment:
We should probably use `accountTwo` here (the non-issuer account), so it's a little more realistic.